emote/memory/table.py

```diff
@@ -36,12 +61,16 @@ def add_sequence(self, identity: int, sequence):
         """add a fully terminated sequence to the memory"""
         ...
 
-    def store(self, path: str) -> bool:
+    def store(
+        self, path: str, version: TableSerializationVersion | None = None
```

Comment:
why do we need to allow this being None? Cant we default to LATEST?

Reply:
(same comment for restore)

Reply:
For restore there's a semantic difference (override vs "use the file"). Here, yes. Good catch 👍

Reply:
Right of course, sorry that makes sense. Nice.